docs/modules/table.md

```diff
@@ -192,7 +192,7 @@ quill.table.deleteColumn();
 
 #### deleteTable
 
-Delete the table at the current selection range.
+Deletes the table at the current selection range.
```

Comment:
Why do you need to add 'current' everywhere? Do you need to highlight the difference between a previous selection?

Reply:
There's an important thing. Quill operates with lots of selection ranges simultaneously, and users should understand that in this case the table will be deleted at the range that is declared inside this function.

Reply:
@DokaRus What's your opinion? Is it important that we highlight that it is the current range or we can just leave 'at the selection range'?

Reply:
'at the selection range' LGTM